docs/api_v0.yml

```diff
@@ -1760,6 +1761,29 @@ tags:
     description: User or organization profile images
 
 paths:
+  /admin/configs/all:
```

Comment:
@rhymes I'd appreciate your help in constructing the spec properly, and what I should define.

I think in general we shouldn't make promises about the individual attributes of `SiteConfig` because we should maintain the freedom to change this a little more liberally right now.

Reply:
yeah, I think this should definitely treat configuration as a unified object, not treat single keys individually, see my comment in the route file